Ludown/lib/toLU-helpers.js

```diff
@@ -16,20 +16,20 @@ const toLUHelpers = {
         let fileContent = '';
         let luisObj = new helperClasses.rLuisObj();
         let intentInObj;
-        if(LUISJSON.intents.length >= 0) {
+        if(LUISJSON.intents && LUISJSON.intents.length >= 0) {
```

Comment:
This can optionally be simplified with `if (Array.isArray(LUISJSON.intents))...`